README.md

```diff
@@ -61,31 +57,27 @@ After you've entered the auto code, the credentials are stored in *gdrive_temp/.
 
 
 
-### Deploying the application
+## Deploying the application
```

Comment:
@sabre1041 shouldn't this section be part of `## Deploying to OpenShift`?

Reply:
Sure. Will update based on other feedback